priam/src/main/java/com/netflix/priam/cluster/management/Flush.java

```diff
@@ -16,75 +16,83 @@
 package com.netflix.priam.cluster.management;
```

Comment:
It would be easier to review if you separate noncompaction related changes with this PR.

Reply:
This was per design. Flush and compaction both extend IClusterManagement. This was done so we don't have repeat of code.

Reply:
Correct, but the refactoring of IClusterManager could have been in a different commit.